lib/cfme-versions.cr

```diff
@@ -0,0 +1,92 @@
+module CFME
+  class Versions
+    FIELDS   = [
+         "ManageIQ",      "",       "CFME",  "CloudForms", "CP4MCM", "Ruby", "Rails", "PostgreSQL"
+    ]
+    VERSIONS = [
+      %w[ N/A             N/A        5.1.z    2.0           N/A       N/A     N/A      N/A        ],
+      %w[ N/A             N/A        5.2.z    3.0           N/A       N/A     N/A      N/A        ],
+      %w[ Anand           1.y.z      5.3.z    3.1           N/A       N/A     N/A      N/A        ],
+      %w[ Botvinnik       2.y.z      5.4.z    3.2           N/A       N/A     N/A      N/A        ],
+      %w[ Capablanca      3.y.z      5.5.z    4.0           N/A       2.2.z   4.2.z    9.4.z      ],
+      %w[ Darga           4.y.z      5.6.z    4.1           N/A       2.2.z   5.0.z    9.4.z      ],
+      %w[ Euwe            5.y.z      5.7.z    4.2           N/A       2.3.z   5.0.z    9.5.z      ],
+      %w[ Fine            6.y.z      5.8.z    4.5           N/A       2.3.z   5.0.z    9.5.z      ],
+      %w[ Gaprindashvili  7.y.z      5.9.z    4.6           N/A       2.3.z   5.0.z    9.5.z      ],
+      %w[ Hammer          8.y.z      5.10.z   4.7           N/A       2.4.z   5.0.z    9.5.z      ],
+      %w[ Ivanchuk        9.y.z      5.11.z   5.0           1.2,1.3   2.5.z   5.1.z    10.y       ],
+      %w[ Jansa           10.y.z     N/A      N/A           2.0       2.5.z   5.2.z    10.y       ],
+      %w[ Kasparov        11.y.z     N/A      N/A           N/A       2.6.z   5.2.z    10.y       ]
+    ]
```

Comment:
I wonder what `crystal tool format` will do to this (many editor implementations run this automatically on save, btw)

Reply:
This was apparently a harder question to answer than I was expecting...

Ran the following:

```console
$ crystal tool format lib/cfme-versions.cr
$ git diff
# nothing changed
```

Thought that was surprising (my crystal writing can't be "THAT" good...

Ran it with `--check`... nothing.

And then proceeded to go on a wild goose chase to figure out where this is implemented... with still no answer.

Unless it won't change it, then in that case, "it won't change a thing".